lib/cw-core/src/tile.rs

```diff
@@ -88,9 +92,18 @@ impl Tile {
             return StatusCode::ErrorInternal;
         }
 
+        let different_frame_buffer = get_different_framebuffer(&mut self.previous_framebuffer,frame_buffer_unordered);
+
+        //check to use command 2 or command 3
+        if different_frame_buffer.len() / 5 < 100 {
```

Comment:
Leave a comment which explains the value 100, and also why you might want to change it in the future.